docs/reference/commandline/images.md

```diff
@@ -52,6 +52,39 @@ uses up the `VIRTUAL SIZE` listed only once.
     postgres                  9.3.5               746b819f315e        4 days ago          213.4 MB
     postgres                  latest              746b819f315e        4 days ago          213.4 MB
 
+### Listing images by name and tag
+
```

Comment:
@moxiegirl thanks :+1: ! I tend to use *future* more than I should too (even in my native language so :sweat:)
@thaJeztah it took me a moment to interpret it correctly too :stuck_out_tongue_closed_eyes:.